config/routes.rb

```diff
@@ -1,8 +1,19 @@
 Rails.application.routes.draw do
+
+  namespace :api do
+    namespace :v1 do
+      namespace :items do
+        get 'find',     to: "find#show",  defaults: {format: :json}
+        get 'find_all', to: "find#index", defaults: {format: :json}
+      end
+    end
+  end
+
```

Comment:
like you were saying earlier, let's talk about ways that we can DRY up the routes